Assets/HoloToolkit/Utilities/Scripts/TextToSpeechManager.cs

```diff
@@ -29,9 +56,13 @@ public class TextToSpeechManager : MonoBehaviour
         [Tooltip("The audio source where speech will be played.")]
         public AudioSource audioSource;
 
+        [Tooltip("The voice that will be used to generate speech.")]
+        public TextToSpeechVoice voice;
```

Comment:
Ahh. I see. I've spent all these years only using fields for private use and only exposing properties. Therefore I've been in the habit of making fields camel case.

I'm happy to fix this, but how do you guys recommend I proceed? I can fix just the Voice field or I can fix all of them. If I fix all of them, what are the potential consequences with Unity serialization? For example, any existing project that has TextToSpeechManager in the scene will have the Audio Source serialized to audioSource. Does Unity handle the case change or will it break the scene? And if it breaks, are we OK with that?

Let me know what you think and I'll change it.